docs/minio-client-complete-guide.md

```diff
@@ -889,19 +890,20 @@ USAGE:
    mc rm [FLAGS] TARGET [TARGET ...]
 
 FLAGS:
-  --versions                    remove object(s) and all its versions
-  --rewind value                revert to older version of object(s) in time
-  --version-id value            delete a specific version of an object
-  --recursive, -r               remove recursively
-  --force                       allow a recursive remove operation
-  --dangerous                   allow site-wide removal of objects
-  --incomplete, -I              remove incomplete uploads
-  --fake                        perform a fake remove operation
-  --stdin                       read object names from STDIN
-  --older-than value            remove objects older than L days, M hours and N minutes LMN[d|h|m]. (default: 0)
-  --newer-than value            remove objects newer than L days, M hours and N minutes LMN[d|h|m]. (default: 0)
-  --encrypt-key value           encrypt/decrypt objects (using server-side encryption with customer provided keys)
-  --help, -h                    show help
+  --versions                       remove object(s) and all its version(s)
```

Comment:
```suggestion
  --versions                       remove object(s) and all its versions
```

Reply:
It is on purpose `--versions` lists `a version` or `versions`

Reply:
Yeah, but ` remove object(s) and all its version` doesn't make sense grammatically.